internal/pkg/inject/offset_results.json

```diff
@@ -864,7 +864,9 @@
                       "0.10.0",
                       "0.11.0",
                       "0.12.0",
-                      "0.13.0"
+                      "0.13.0",
+                      "1.25.0",
```

Comment:
That is strange. These offsets are cached and should not have been re-evaluated. This is the reason for the `otelglobal` test failing.

Reply:
Ran it locally without the cache and it didn't produce this diff.

Reply:
using the cached version fixed it, so there is either an issue with my local setup or a problem with generating the offsets for otelglobal

Reply:
@RonFed ah didn't see your comment, must be something on my end then. Will try to look into it

Reply:
@RonFed sorry but just to check, did you also try deleting the `.tools` directory to re-build offsetgen when you tested it?

Reply:
yes

Reply:
Update: I ran this a couple more times from "fresh" and still got the diff, but it looks like clearing my docker image cache fixed it. Maybe something was up in one of the images I had pulled from a previous run